src/main/java/com/synopsys/integration/alert/database/security/StringEncryptionConverter.java

```diff
@@ -39,7 +39,14 @@
     private static EncryptionUtility encryptionUtility;
 
     public boolean isInitialized() {
-        return StringEncryptionConverter.encryptionUtility != null;
+        boolean initialized = StringEncryptionConverter.encryptionUtility != null;
+        if (initialized) {
+            initialized = encryptionUtility.isInitialized();
+        } else {
+            initialized = false;
+        }
+
+        return initialized;
```

Comment:
This could be one line:

return StringEncryptionConverter.encryptionUtility != null && encryptionUtility.isInitialized();